docs/schemas/resource/IfcPresentationAppearanceResource/Entities/IfcTextureCoordinateIndices.md

```diff
@@ -0,0 +1,13 @@
+# IfcTextureCoordinateIndices
+
+The _IfcTextureCoordinateIndices_ provide the texture coordinates for an _iFCIndexedPolygonalFace_. The list of _TexCoordIndex_ holds the indices into the _IfcTextureVertexList_ that correspond to the list of _CoordIndex_ at _TexCoordsOf_.
```

Comment:
>"The list of _TexCoordIndex_"

The list provided at attribute _TexCoordIndex_?

Otherwise people might confuse it with the element type of the list?